src/modules/admissions/ApplicantStatusBadge.tsx

```diff
@@ -11,10 +12,10 @@ const Wrapper = styled.div`
 const handleStatusLabelText = (status: ApplicantStatus) => {
```

Comment:
Can probably be same parsing helper as the other. Maybe even just a translate helper